python/python/bystro/api/proteomics.py

```diff
@@ -34,13 +34,36 @@ class DatasetTypes(Enum):
 
 
 def upload_proteomics_dataset(
-    protein_abundance_file: str,
-    experiment_annotation_file: str | None = None,
-    annotation_job_id: str | None = None,
-    proteomics_dataset_type: DatasetTypes = DatasetTypes.fragpipe_TMT,
-    user_dir: str = DEFAULT_DIR,
-    print_result: bool = True,
-) -> dict[str, Any]:
+        protein_abundance_file: str,
+        experiment_annotation_file: str | None = None,
+        annotation_job_id: str | None = None,
+        proteomics_dataset_type: DatasetTypes = DatasetTypes.fragpipe_TMT,
+        print_result: bool = True,
+    ) -> dict[str, Any]:
+    """
+    Upload a fragpipe-TMT dataset through the /api/jobs/proteomics/ endpoint and 
+    update the annotation job.
+
+    Parameters
+    ----------
+    protein_abundance_file : str
+        Path to the protein abundance file.
+    experiment_annotation_file : str | None
+        Path to the experiment annotation file.
+    experiment_annotation_file : str | None
+        ID of the job associated with the annotation dataset.
+    proteomics_dataset_type : DatasetTypes
+        Type of the proteomics dataset (we only support fragpipe-TMT currently).
+    bystro_credentials_dir : str
```

Comment:
Credentials dir no longer needed

Reply:
@dlin30 the bystro_credentials_dir mentions need to be ripped out from docstrings